Commands/ascii.py

```
@@ -137,4 +157,48 @@ def reply_with_ascii(bot, message):
+    try:
+        image = Image.open(BytesIO(img_bytes))
+        frames = []
+        if getattr(image, "is_animated", False):
+            total_frames = image.n_frames
+            max_frames = 20
+
+            if total_frames > max_frames:
+                # Calculate the interval to sample frames
+                interval = total_frames // max_frames
+            else:
+                interval = 1
```

Ruff

Use ternary operator `interval = total_frames // max_frames if total_frames > max_frames else 1` instead of `if`-`else`-block

(SIM108)